crates/anvil/src/lib.rs

```diff
@@ -334,44 +334,51 @@ impl NodeHandle {
         ProviderBuilder::new(&self.config.get_ipc_path()?).build().ok()
     }
 
-    /// Signer accounts that can sign messages/transactions from the EVM node
+    /// Signer accounts that can sign messages/transactions from the EVM node.
     pub fn dev_accounts(&self) -> impl Iterator<Item = Address> + '_ {
         self.config.signer_accounts.iter().map(|wallet| wallet.address())
     }
 
-    /// Signer accounts that can sign messages/transactions from the EVM node
+    /// Signer accounts that can sign messages/transactions from the EVM node.
     pub fn dev_wallets(&self) -> impl Iterator<Item = PrivateKeySigner> + '_ {
         self.config.signer_accounts.iter().cloned()
     }
 
-    /// Accounts that will be initialised with `genesis_balance` in the genesis block
+    /// Accounts that will be initialised with `genesis_balance` in the genesis block.
     pub fn genesis_accounts(&self) -> impl Iterator<Item = Address> + '_ {
         self.config.genesis_accounts.iter().map(|w| w.address())
     }
 
-    /// Native token balance of every genesis account in the genesis block
+    /// Native token balance of every genesis account in the genesis block.
     pub fn genesis_balance(&self) -> U256 {
         self.config.genesis_balance
     }
 
-    /// Default gas price for all txs
+    /// Default gas price for all txs.
     pub fn gas_price(&self) -> u128 {
         self.config.get_gas_price()
     }
 
-    /// Returns the shutdown signal
+    /// Returns the shutdown signal.
     pub fn shutdown_signal(&self) -> &Option<Signal> {
         &self._signal
     }
 
-    /// Returns mutable access to the shutdown signal
+    /// Returns mutable access to the shutdown signal.
     ///
-    /// This can be used to extract the Signal
+    /// This can be used to extract the Signal.
     pub fn shutdown_signal_mut(&mut self) -> &mut Option<Signal> {
         &mut self._signal
     }
 
-    /// Returns the task manager that can be used to spawn new tasks
+    /// Fires shutdown signal.
+    ///
+    /// Called by long-running tests to make sure anvil instance is terminated.
+    pub fn fire_shutdown_signal(&mut self) {
+        self._signal.take().map(|signal| signal.fire());
```

Comment:
unwrap in the map?

Reply:
added in https://github.com/foundry-rs/foundry/pull/8947/commits/654b1590c4cbfb17b78b8a51e79668787bdc0421
also removed testing `DAPP_REMAPPINGS` from env var in cli tests https://github.com/foundry-rs/foundry/pull/8947/commits/98873ec74891ab28bc24341ed7275f6662b8a852, we have test here that uses jail so should be covered https://github.com/foundry-rs/foundry/blob/c9d7b48fb0cdddc33c61db82fc3a94dd7e602c9e/crates/config/src/lib.rs#L3086